app/assets/stylesheets/fae/modules/forms/_label.scss

```diff
@@ -18,4 +18,19 @@ label {
     font-style: italic;
     line-height: 1.3;
   }
+
+  &.label--two_col {
+    display: flex;
```

Comment:
Ohhhh lanta are we supporting flex now??? Depending on the spec you guys are working on, the current Fae supports IE 9. Shockingly, I vote to leave it that way and go full ham on flex in version 2 along with the rest of the breaking changes that v2 promises in the front-end.